server/node/inventory.js

```diff
@@ -77,14 +37,18 @@ const productsExist = productList => {
   }, !!productList.data.length);
 };
 
-exports.orders = {
-  create: createOrder,
-  retrieve: retrieveOrder,
-  update: updateOrder,
+// Shipping cost.
+const getShippingCost = id => {
```

Comment:
this confuses me a bit. what's the id here? is it just `free` or `express` ? a better param name might be `shippingType` and instead of using an object this way why don't we just use: 
`` `
return shippingType === 'free' ? 0 : 500;
```
or an if statement, I feel like using an object this way is a bit of an anti-pattern since we never use the object again.

Reply:
My thinking was to mimic the `shippingoptions` that are used in the payment request: https://github.com/stripe/stripe-payments-demo/pull/48/files#diff-51e143a26fdd4bb09395de2ec4ae9ce6R151

In a real life scenario these are probably retrieved from a database or API. For Node I could pull it into the config.js and serve it to the frontend via the `/config` backend, but for the python server we don't have a config. Any idea what would be the right path forward for the python server?

Reply:
Gotcha. I'm peaceful with keeping that as a hardcoded object then as long as we have a comment explaining. Something like ```// These shipping options would be loaded from your database```